sdk/kfp-tekton-limitation.md

```diff
@@ -0,0 +1,9 @@
+
+## kfp-tekton limitation 
+- Please be aware that defined Affinity, Node Selector, and Tolerations are applied to all the tasks in the same pipeline because there's only one podTemplate allowed in each pipeline.
+
+- When you add test cases to compiler_tests, the output of pipeline/pipelinerun yaml may has uncertain values or orders, then you can define a lambda function as normalize_compiler_output_function to pass the testing.
```

Comment:
This will be not needed since i already have a fix in the volume op pr.

Reply:
ok